Posts/2021/03/DarwinJS-Contribution-LightningFastandEasyProvisioningofGitwithSSHKeyAuthenticationonWindows.md

```diff
@@ -0,0 +1,117 @@
+---
+post_title: Lightning Fast and Easy Provisioning of Git with SSH Key Authentication on Windows
+username: DarwinJS@sanoys.com
+Catagories: PowerShell
+tags: Git, DevOps
+Summary: Getting your team setup with Git over SSH as quickly as possible!
+featured_image: ./media/DarwinJS-Contribution-LightningFastandEasyProvisioningofGitwithSSHKeyAuthenticationonWindows/windows-git-ssh.png
+CanonicalURL: https://missionimpossiblecode.io/post/lightning-fast-and-easy-provisioning-of-git-with-ssh-key-authentication-on-windows/
```

Comment:
This metadata value will be ignored by the system, but I can use it to configure the canonical URL before publishing.

Reply:
Yes that was my intent!